pinax/stripe/actions/customers.py

```diff
@@ -151,8 +155,16 @@ def sync_customer(customer, cu=None):
         customer: a Customer object
         cu: optionally, data from the Stripe API representing the customer
     """
+    if customer.date_purged is not None:
+        return
+
     if cu is None:
         cu = customer.stripe_customer
+
+    if cu.get('deleted', False):
```

Comment:
Just a small thing, but can you change to double-quotes to keep consistent with project style?  Thanks!

Reply:
@paltman np, done!